mmv1/templates/terraform/examples/looker_instance_enterprise_full.tf.erb

```diff
@@ -0,0 +1,90 @@
+resource "google_looker_instance" "<%= ctx[:primary_resource_id] %>" {
+  name               = "<%= ctx[:vars]["instance_name"] %>"
+  platform_edition   = "LOOKER_CORE_ENTERPRISE_ANNUAL"
+  region             = "us-central1"
+  private_ip_enabled = true
+  public_ip_enabled  = true
+  reserved_range     = "${google_compute_global_address.looker_range.name}"
+  consumer_network   = "projects/${data.google_project.project.number}/global/networks/${data.google_compute_network.looker_network.name}"
+  admin_settings {
+    allowed_email_domains = ["google.com"]
+  }
+  encryption_config {
+    kms_key_name = google_kms_crypto_key.looker_crypto_key.id
+  }
+  maintenance_window {
+    day_of_week = "THURSDAY"
+    start_time {
+      hours   = 22
+      minutes = 0
+      seconds = 0
+      nanos   = 0
+    }
+  }
+  deny_maintenance_period {
+    start_date {
+      year = 2050
+      month = 1
+      day = 1
+    }
+    end_date {
+      year = 2050
+      month = 2
+      day = 1
+    }
+    time {
+      hours = 10
+      minutes = 0
+      seconds = 0
+      nanos = 0
+    }
+  }
+  oauth_config {
+    client_id = "<%= ctx[:vars]["client_id"] %>"
+    client_secret = "<%= ctx[:vars]["client_secret"] %>"
+  }
+  depends_on   = [
+    google_service_networking_connection.looker_vpc_connection,
+    google_kms_crypto_key.looker_crypto_key
+  ]
+}
+
+resource "google_service_networking_connection" "looker_vpc_connection" {
+  network                 = data.google_compute_network.looker_network.id
+  service                 = "servicenetworking.googleapis.com"
+  reserved_peering_ranges = [google_compute_global_address.looker_range.name]
+}
+
+resource "google_compute_global_address" "looker_range" {
+  name          = "<%= ctx[:vars]["address_name"] %>"
+  purpose       = "VPC_PEERING"
+  address_type  = "INTERNAL"
+  prefix_length = 20
+  network       = data.google_compute_network.looker_network.id
+}
+
+data "google_project" "project" {}
+
+// This example assumes this network already exists.
```

Comment:
We'll want to create the network in the example itself, so the test works!

Reply:
Can this network not be constructed inside the running project?

These are the examples I'm seeing:
mmv1/templates/terraform/examples/vertex_ai_endpoint_network.tf.erb
mmv1/templates/terraform/examples/redis_instance_full.tf.erb

Reply:
Vertex's test is skipped: https://github.com/GoogleCloudPlatform/magic-modules/blob/499e9487546f0926a6c364afb509d8bad4fc9a32/mmv1/products/vertexai/Endpoint.yaml#L51-L53

Redis uses a bootstrapped network due to service network quota reasons: https://github.com/GoogleCloudPlatform/magic-modules/blob/499e9487546f0926a6c364afb509d8bad4fc9a32/mmv1/products/redis/Instance.yaml#L51-L52

The same constraints that are true for Redis aren't true here, so we should provision a network inline to avoid burning one of the 25 global network quotas the project has permanently on a bootstrapped one.

Reply:
Done!